aiohttp/client_reqrep.py

```diff
@@ -7,6 +7,7 @@
 import sys
 import traceback
 import warnings
+from collections.abc import Mapping as ABCMapping
```

Comment:
Don't rename it. The typing one is deprecated anyway.

Reply:
https://github.com/aio-libs/aiohttp/pull/11088